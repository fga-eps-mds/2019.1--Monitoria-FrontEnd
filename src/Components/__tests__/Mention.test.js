import React from 'react';
import Mention from '../Feedback/Mention';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing Mention component', () => {
    it('Test if Mention renders correctly', () =>{
        const tree = shallow(<Mention/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});