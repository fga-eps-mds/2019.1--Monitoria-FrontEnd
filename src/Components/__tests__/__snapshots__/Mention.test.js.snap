// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Mention component Test if Mention renders correctly 1`] = `
<SimpleSelect
  classes={
    Object {
      "formControl": "SimpleSelect-formControl-2",
      "root": "SimpleSelect-root-1",
      "selectEmpty": "SimpleSelect-selectEmpty-3",
    }
  }
/>
`;
